src/pages/index.js

```diff
@@ -90,19 +185,87 @@ function openEditCardModal() {
   addCardModal.open();
 }
 
+function openProfileImageModal() {
+  editAvatarModal.open();
+}
+
 /*------------------------------------------------------------------*/
-/*                          Event Objects                            */
+/*                          EVENT HANDLERS                            */
 /*------------------------------------------------------------------*/
 
 function handleProfileEditSubmit({ title, description }) {
-  userInfo.setUserInfo({ title, description });
-  profileEditModal.close();
+  profileEditModal.setLoading(true);
+  api
+    .updateProfileInfo({ title, description })
+    .then((res) => {
+      profileUserInfo.setUserInfo({ title, description });
+      profileEditModal.close();
+    })
+    .finally(() => profileEditModal.setLoading(false));
+}
+
+function likeCard(card) {
+  api
+    .likeCard(card.getId())
+    .then(() => {
+      card._handleLikeIcon();
```

Comment:
`_handleLikeIcon`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.


**Please, correct it everywhere**